exercise3/block.py

```diff
@@ -31,14 +31,14 @@ def __init__(
     def hash(self) -> bytes:
         """
         TODO: Oblicz hash bloku wykorzystując do tego funkcję `hash` z modułu simple_cryptography.
-            Hash powinien składać się z wszystkich składowych bloku:
+            Hash powinien zostać obliczony ze skonkatenowanych składowych bloku:
```

Comment:
To jest prawda, ale może należy napisać po prostu `zsumowanych` czy coś? Wtedy jest trochę jaśniej że te bytes można sobie po prostu pododawać.

Reply:
W ex1 też mówimy o konkatenacji a nie o sumowaniu.

Reply:
To może tam należy też zmienić. Nie jestem w 100% przekonany, ale prostszy język w poleceniach może być dobry 

cc @war-in